packages/icon/src/Icon.spec.tsx

```diff
@@ -79,7 +79,7 @@ describe('packages/Icon/createGlyphComponent createGlyphComponent()', () => {
   });
 
   test('The function returned by createGlyphComponent has the displayName: "MyGlyph"', () => {
-    expect(GlyphComponent.displayName).toBe('MyGlyph');
+    expect(GlyphComponent.displayName).toBe('GlyphMyGlyph');
```

Comment:
We may need to add some documentation around custom glyph display names being used with `IconButton` if our tests needed to be updated here.

Reply:
What do you think should be documented?

Reply:
I suppose, since we're relying on "Glyph" being the prefix to any glyph display name in places, we should document something around that.

Reply:
Sounds good, I added as a note to the readme.